get_item_ranges.py

```
@@ -0,0 +1,50 @@
+import json
+
+def get_ranges():
+    with open('/tmp/telemetry.rs', 'r') as f:
+        lines = f.readlines()
+
+    blocks = []
+    current_block = None
+    nesting = 0
+
+    for i, line in enumerate(lines):
+        line = line.strip()
```

Ruff (0.15.9)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)